agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -95,10 +96,26 @@ function collect_sos {
 	fi
 }
 
-collect_kernel_config &
-collect_libvirt &
-collect_topology &
-collect_sos &
-collect_block &
+if [[ -n "$sysinfo" ]]; then
+	for item in ${sysinfo//,/ };do
+        	if [[ "$item" == "default" ]] || [[ "$item" == "all" ]]; then
+			collect_kernel_config &
```

Comment:
default and all options are allowed. we just have pass default if we want to collect default info and all if we want to collect everything. we don't have to covert them to appropriate lists.